src/pages/spawn/spawnHost/editSpawnHostModal/useEditSpawnHostModalState.ts

```diff
@@ -0,0 +1,67 @@
+import { useReducer } from "react";
+import { InstanceTag } from "gql/generated/types";
+import { MyHost } from "types/spawn";
+
+interface editSpawnHostState {
+  expiration?: Date;
+  noExpiration: boolean;
+  displayName?: string;
+  instanceType?: string;
+  volumeId?: string;
+  addedInstanceTags?: InstanceTag[];
+  deletedInstanceTags?: InstanceTag[];
+}
+
+export const useEditSpawnHostModalState = (host: MyHost) => ({
+  reducer: useReducer(reducer, host, init),
+  defaultEditSpawnHostState: init(host),
+});
+
+const init = (host: MyHost) => ({
+  displayName: host.displayName,
+  expiration: host.expiration,
+  noExpiration: host.noExpiration,
+  instanceType: host.instanceType,
+  volumeId: "",
```

Comment:
I think we need to set the volumeId to what exists in host. Same with the instance tags so the user can the initial values.

Reply:
It looks like we don't actually have to do the same thing with instance tags because the UserTagsField component takes the instance tags from the `MyHost` as the initial display

Reply:
Yup